src/core/gpu.js

```diff
@@ -83,7 +83,7 @@ class GPU extends GPUCore {
 		if (typeof fn === 'undefined') {
 			throw 'Missing fn parameter';
 		}
-		if (!utils.isFunction(fn)) {
+		if (!utils.isFunction(fn) && typeof fn !== 'string') {
```

Comment:
The logic that checks for `isFunction` should probably contain the logic for string or not.  Can you more that there?

Reply:
No, if fn is a string, that will fire the if and throw an error